### DISCUSSION TITLE
implement Homeassistant autodiscovery

### DISCUSSION THREAD

**Discussion by dcboy87** [jomjol/AI-on-the-edge-device#1132] [Ideas]
Since this project is used in a lot of homeassistant smart homes it would be great to have the homeassistant autodiscovery feature. In this issue (https://github.com/jomjol/AI-on-the-edge-device/issues/299#issuecomment-993448517) @Henry-Sir has written his own discovery, maybe this is a starting point for @jomjol to integrate this feature

https://www.home-assistant.io/docs/mqtt/discovery/

Thanks a lot for this project!

**Comment by caco3** [comment]
Good idea, I added https://github.com/jomjol/AI-on-the-edge-device/issues/1133 to track it
